.github/workflows/dockerhub.yml

```diff
@@ -18,34 +18,43 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
-
+          
+      - name: Set Environment Variable for Either Master or Develop
+        run: |
+          if [ "${{ github.ref }}" = "refs/heads/develop" ]; then
+            echo "ENV_VAR=dev" >> $GITHUB_ENV
+          elif [ "${{ github.ref }}" = "refs/heads/master" ]; then
+            echo "ENV_VAR=candidate" >> $GITHUB_ENV

```

Comment:
Better name. Branch name or something. I am not sure I understand this. Why would we push master as a candidate branch

Reply:
you are right. I got confused the release with candidate stuff. I reverted my change

Reply:
However, I forced the ref_name to develop because we don't create images with master tag